### PR TITLE
Poll to ensure that element becomes a stylesheet

### LINKED ISSUE #27: document.styleSheets is not reliably updated
`document.styleSheets` not always synchronously updated with a new element's sheet.

The element added [here](https://github.com/tim-evans/ember-autoresize/blob/master/addon/system/font-loaded.js#L15) may not actually cause a sheet to be added and found in `document.styleSheets` [here](https://github.com/tim-evans/ember-autoresize/blob/master/addon/system/font-loaded.js#L18).

This presents as an error:

### PR DESCRIPTION
**Pull request by mixonic**
Under memory pressure or some other case, Chrome will fail to synchronously update the document.styleSheets list. This adds a polling step that confirms stylesheets are loaded correctly.

Fixes #27

### CODE REVIEW

**Comment by tomdale** [comment]
>  or some other case

Do you know what the other case(s) are?

**Comment by tim-evans** [comment]
I'm going to merge this, but would love to continue this conversation. I've seen this reported in another instance here: https://github.com/tim-evans/ember-plupload/issues/46

Which seems to be directly related to this ticket (albeit written in a different library)

File: addon/system/font-loaded.js [hunk 1]
```diff
@@ -30,6 +25,34 @@ function injectAdobeBlank() {
   }
```
Comment (mixonic) [inline addon/system/font-loaded.js]: Uh oh, this should have a `return true` at the end or the polling will never stop.

Comment (tim-evans) [inline addon/system/font-loaded.js]: :cold_sweat:

**Pull request merged**